src/pokemon-service/repository/pokemon_test.go

```diff
@@ -0,0 +1,136 @@
+package repository
+
+import (
```

Comment:
For all the test files, it would be cleaner if you use assertions and mocks. Take a look to these packages:
"github.com/stretchr/testify/assert"
"github.com/stretchr/testify/mock"

Reply:
Done

Reply:
I'm not sure if it is out of the scope I avoid to use external packages to show how I would do it as a starter, but I'll take a look  into those. unless you suggest to move into that direction

Reply:
Please let me know what should I do , I won't mark it as resolved for the moment